### PR TITLE
move ember-deep-tracked dep to showcase

### PR DESCRIPTION
**Pull request by Dhaulagiri**
### :pushpin: Summary

AFAICT that is only used in the showcase but was moved to a component dep in https://github.com/hashicorp/design-system/commit/a7206c8071966b275b41fb6aa014c519a5d1c1f6

### CODE REVIEW

**Review by alex-ju — APPROVED** [review]
My bad, I stumbled upon it at rebase, but for some reason I thought it was used inside the Table component

File: .changeset/silver-masks-change.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+remove `ember-deep-tracked` dependency that was not used
```
Comment (alex-ju) [inline .changeset/silver-masks-change.md:5]:
```suggestion
Remove `ember-deep-tracked` dependency that was not used
```

**Comment by alex-ju** [comment]
Also [hard to spot in the diff](packages/components/package.json) – thanks for catching that!

**Comment by Dhaulagiri** [comment]
Indeed, I obviously didn't notice it when I had reviewed it 👀 . Just something I stumbled across poking around in the codebase this morning.

**Pull request merged**